server.js

```diff
@@ -6,4 +6,8 @@ const server = app.listen(PORT, () => {
   console.log(' Tenthop server is listening on port 3000')
 })
 
-app.use(express.static('./client'))
\ No newline at end of file
+app.use(express.static('./client'))
+
+app.get('*', (req, res) => {
```

Comment:
It looks like this is the magic that we use to request your data with your router! Si o no?